types/mailparser/package.json

```diff
@@ -0,0 +1,6 @@
+{
+	"private": true,
+	"dependencies": {
+	  "iconv-lite": "^0.6.3"
+	}
+  }
```

Comment:
```suggestion
}
```

Reply:
Fixing.  Don't remember which type I copied it from 🙁